ecs-logs-collector.sh

```diff
@@ -310,6 +311,16 @@ get_iptables_info() {
   ok
 }
 
+get_open_files() {
+  try "get open files list"
+
+  mkdir -p "$info_system"
+  for d in /proc/*/fd; do echo "$d"; find "$d" -maxdepth 1 | wc -l; done > "$info_system"/open-file-counts.txt
+  ls -l /proc/*/fd > "$info_system"/open-file-details.txt
```

Comment:
Does this require root permission? (Sorry not sure if log collector runs with root permission or not)

Reply:
yep, logs collector always runs with root permission, the script exits on startup if it doesnt have it